.travis/run_travis_job.sh

```diff
@@ -15,26 +15,25 @@ if [ "$CHECK_RUST" = true ]
 then
     # Install cargo-audit if it's not already.
     cargo audit --version || cargo install cargo-audit --force
-    # Install nightly rust version and clippy.
-    rustup toolchain install $RUST_NIGHTLY_VERSION
-    cargo +$RUST_NIGHTLY_VERSION clippy -V | grep $RUST_CLIPPY_VERSION || cargo +$RUST_NIGHTLY_VERSION install clippy --vers $RUST_CLIPPY_VERSION --force
-    # TODO: use stable rust everywhere when ECR-1839 fixed.
-    # Install the stable toolchain and rustfmt.
-    rustup toolchain install stable
-    rustup component add rustfmt-preview --toolchain stable
-    rustup run stable rustfmt -V
+    # Install clippy and rustfmt.
+    rustup component add clippy
+    rustup component add rustfmt
+    rustfmt -V
+    cargo clippy -V
 
     echo 'Performing checks over the rust code'
     cd "${EJB_RUST_BUILD_DIR}"
     # Check the formatting.
-    cargo +stable fmt --all -- --check
+    cargo fmt -- --check
 
     # Run clippy static analysis.
     # TODO Remove when clippy is fixed https://github.com/rust-lang-nursery/rust-clippy/issues/2831
     # Next 2 lines are a workaround to prevent clippy checking dependencies.
-    cargo +${RUST_NIGHTLY_VERSION} check
-    cargo +${RUST_NIGHTLY_VERSION} clean -p java_bindings
-    cargo +${RUST_NIGHTLY_VERSION} clippy --all --tests --all-features -- -D warnings
+#    cargo +${RUST_STABLE_VERSION} check
+#    cargo +${RUST_STABLE_VERSION} clean -p java_bindings
+#    cargo +${RUST_STABLE_VERSION} clippy --all --tests --all-features -- -D warnings
+# TODO: Commented until ECR-2755 is fixed
+#    cargo clippy --all --tests --all-features -- -D warnings
```

Comment:
Do we have to skip clippy (the issue in todo seems to be resolved)? Or are there other issues?